docs/tutorial/02_deploying_magma_orchestrator.md

```diff
@@ -0,0 +1,225 @@
+# 2. Deploying Magma Orchestrator
+
+In this section, we will deploy Magma Orchestrator on AWS's managed Kubernetes service (EKS) using Juju.
+
+!!! note
+    :material-dns: The following steps assume that you have a domain name registered with a DNS provider and that you have a hosted zone in AWS's Route53 associated with this domain. Everywhere you see `<your domain name>` in the following steps, you should replace it with your domain name.
+
+## Create a Kubernetes cluster
+
+Create a Kubernetes cluster on AWS using `eksctl`:
+
+```console
+eksctl create cluster --name magma-orc8r --region us-east-2 --node-type t2.xlarge
```

Comment:
Are you not missing the setup for the EBS CSI to enable storage?

Reply:
Humm well this has been enough for me. `eksctl` uses cloudformation and does "all the magic needed" to have a working Kubernetes cluster

Reply:
Ok, that is interesting, I think the issue must be specific to bootstrapping a controller on EKS then.

Reply:
You were right all along, I'm not sure why it worked in the past, it could be due to a change in `eksctl`.